packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/getAndroidSplashConfig.ts

```diff
@@ -1,20 +1,21 @@
 import { ExpoConfig } from '@expo/config-types';
 
-export type SplashScreenConfig = {
+export interface AndroidSplashConfig {
   xxxhdpi: string | null;
   xxhdpi: string | null;
   xhdpi: string | null;
   hdpi: string | null;
   mdpi: string | null;
   backgroundColor: string | null;
   resizeMode: 'contain' | 'cover' | 'native';
-};
+  fadeDurationMs: number;
+}
 
 const defaultResizeMode = 'contain';
 
 export function getAndroidSplashConfig(
   config: Pick<ExpoConfig, 'splash' | 'android'>
-): SplashScreenConfig | null {
+): AndroidSplashConfig | null {
```

Comment:
```suggestion
): AndroidSplashConfig {
```

since it will never returns null now